source/ada/lsp-ada_handlers.adb

```diff
@@ -776,4 +796,31 @@ package body LSP.Ada_Handlers is
       return R;
    end Handle_Request;
 
+   -------------------------
+   -- Handle_Notification --
+   -------------------------
+
+   overriding procedure Handle_Notification
```

Comment:
I would like message dispatching code to be in language independent part - in server, outside of ada/ directory. So each language server won't reimplement it.